gravatar-quickeditor/src/main/java/com/gravatar/quickeditor/ui/avatarpicker/AvatarPickerViewModel.kt

```diff
@@ -116,6 +116,7 @@ internal class AvatarPickerViewModel(
                                     addAll(currentState.emailAvatars.avatars.filter { it.imageId != avatar.imageId })
                                 },
                             ),
+                            scrollToIndex = null,
```

Comment:
⚠️ We need to do the same in the Failure case. 

https://github.com/user-attachments/assets/22badcc1-61e0-4f8e-adad-bae7aadbd5bc

Reply:
right... fixed!